### ISSUE TITLE
Support Microsoft Outlook 2016 signature editor

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/nvda#7253]
### Steps to reproduce:
With NVDA, open Outlook 2016.
On the ribbon, go to Insert -> Signature -> Signatures...
If needed, create a new Signature.
Once a new or existing signature is active in this dialog, tab to the Signature editor control (one tab stop before the Okay button).
Note that in Outlook 2016, nothing is announced at all.
In previous versions of Office, the control was announced, however arrowing around the control was not always entirely accurate.

### Expected results
NVDA should announce the control, and allow reading / editing the content like a normal Microsoft Word document

### Technical info:
Office 2016 now provides a native UI Automation implementation for _WwN controls. We should ensure NVDA uses this if available. However, this will involve moving oure existing _WwN support into IAccessible, as it is currently at Window level and will conflict with UIA.
